src/js/pages/catalog/PackageDetailTab.js

```diff
@@ -62,14 +84,13 @@ class PackageDetailTab extends mixin(StoreMixin) {
     this.store_listeners = [
       {
         name: "cosmosPackages",
-        events: ["descriptionError", "descriptionSuccess"],
-        unmountWhen(store, event) {
-          if (event === "descriptionSuccess") {
-            return !!CosmosPackagesStore.get("packageDetails");
-          }
-        },
-        listenAlways: false,
-        suppressUpdate: true
+        events: [
+          "descriptionError",
+          "descriptionSuccess",
+          "listVersionsSuccess",
+          "listVersionsError"
+        ],
+        suppressUpdate: false
```

Comment:
I'm curious why this got changed? The `suppressUpdate` boolean is now false, `unmountWhen()` and `listenAlways` is removed.

Reply:
`suppressUpdate` will force to update React when false.
`unmountWhen` it's called when an event happen.
`listenAlways` remove listener on the store change.